src/modules/graphics/graphics.c

```diff
@@ -992,7 +1007,7 @@ Texture* lovrTextureCreate(TextureInfo* info) {
   lovrCheck(info->depth == 1 || info->type != TEXTURE_2D, "2D textures must have a depth of 1");
   lovrCheck(info->depth == 6 || info->type != TEXTURE_CUBE, "Cubemaps must have a depth of 6");
   lovrCheck(info->width == info->height || info->type != TEXTURE_CUBE, "Cubemaps must be square");
-  lovrCheck(measureTexture(info->format, info->width, info->height, info->depth) < 1 << 30, "Memory for a Texture can not exceed 1GB"); // TODO mip?
+  lovrCheck(measureTexture(info->format, info->width, info->height, info->depth) < MAX_BUFFER_SIZE, "Memory for a Texture can not exceed " MAX_BUFFER_SIZE_STR); // TODO mip?
```

Comment:
I'd prefer if this constant were named something other than `MAX_BUFFER_SIZE` since that makes it seem like it's the maximum size of a buffer when it's really the maximum size of a GPU memory allocation.  Maybe `MAX_GPU_ALLOCATION_SIZE`